deal.II-toolchain/packages/dealii.package

```diff
@@ -10,14 +10,16 @@ BUILDDIR=${BUILD_PATH}/deal.II-${VERSION}
 INSTALL_PATH=${ORIG_INSTALL_PATH}/deal.II-${VERSION}
 
 ################################################################################
-# Please do not change the line below!
+# Please do not change the lines below!
 
 ################################################################################
 #Choose general configuration and components of deal.II
 
 CONFOPTS=" \
 -D CMAKE_BUILD_TYPE=DebugRelease \
 -D DEAL_II_WITH_MPI:BOOL=ON \
+-D MPI_CXX_COMPILER=${CXX} \
+-D MPI_CC_COMPILER=${CC} \
```

Comment:
@gfcas This line should be removed, correct?

Reply:
precisely line 22 here

Reply:
that was a typo, fixed.